controller/backup_target_controller.go

```diff
@@ -433,6 +433,13 @@ func (btc *BackupTargetController) reconcile(name string) (err error) {
 		}
 	}()
 
+	// clean up invalid backup volumes that are created during split-brain
+	// https://github.com/longhorn/longhorn/issues/11154
+	clusterVolumeBVMap, err := btc.cleanupDuplicateBackupVolumes(backupTarget)
```

Comment:
Why do we need the prefix `cluster`?
```suggestion
	clusterVolumeBVMap, err := btc.cleanupDuplicateBackupVolumes(backupTarget)
```